### PR TITLE
fix: typo evaluation script + edge case metrics + resize

### PR DESCRIPTION
**Pull request by charlesmindee**
This PR fixes a typo in evaluate.py and 2 edge cases:
- resize a tf tensor with a 0-dimension
- set iou to 0 when iou matrix is empty

Any feedback is welcome!

### CODE REVIEW

File: scripts/evaluate.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 os.environ["TF_CPP_MIN_LOG_LEVEL"] = "2"
 
 from doctr.utils.metrics import LocalizationConfusion, ExactMatch, OCRMetric
-from doctr.datasets import FUNSD
+from doctr.datasets import FUNSD, SROIE
```
Comment (fg-mindee) [inline scripts/evaluate.py:13]: let's avoid importing it since we don't use it for now

**Review by fg-mindee — COMMENTED** [review]
Only a small import to remove and we're good to merge!

File: scripts/evaluate.py (hunk 1 above)
Comment (charlesmindee) [reply scripts/evaluate.py:13]: ok

**Review by fg-mindee — APPROVED** [review]
Thanks for the fix!

**Pull request merged**